components/bl-collapsiblePanel-component/src/index.js

```diff
@@ -48,10 +54,10 @@ export default function CollapsiblePanelComponent({ component, elRef, eventHandl
   );
 }
 
-function PanelTitle({ title, expanded, onClick }) {
+function PanelTitle({ title, layout, expanded, onClick }) {
   return (
     <div className="panel-title" aria-expanded={ expanded } role="button" onClick={ onClick }>
-      <span className="panel-title-text">{ title }</span>
+      <span className={ cn('panel-title-text', layout === ONELINE ? 'oneline' : 'multiline' ) }>{ title }</span>
```

Comment:
```suggestion
      <span className={ cn('panel-title-text', layout) }>{ title }</span>
```

As i see component is not published, so if it was done for backward compatibility - we didn't need it.